### DISCUSSION TITLE
Label checkbox

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/iTwinUI#1837] [Q&A]
hi I recently switched to the @itwin/itwinui-react" version: "^2.12.21",
when I go to use the checkbox or ToggleSwitch, it inserts the check box above and the label immediately below, I can't keep them on the same line

I attach a small image of what happened
<img width="49" alt="error_checkbox" src="https://github.com/iTwin/iTwinUI/assets/57446652/60d06753-02e1-47a3-b73e-907762fb254c">
<img width="462" alt="html_checkbox" src="https://github.com/iTwin/iTwinUI/assets/57446652/5a7126a0-098b-4eee-b345-a7069fc3859e">

Thanks for your help

**Comment by mayank99** [comment]
You might have custom styles overriding the iTwinUI styles. It looks fine here: https://itwin.github.io/iTwinUI/legacy/v2/react/?path=/story/input-toggleswitch--label-right
